ilc/client.js

```diff
@@ -144,10 +144,20 @@ function getCurrentBasePath() {
     return currentPath.basePath;
 }
 
+function getNormalizedLocation() {
+    // we should respect base tag for cached pages
+    const base = document.querySelector('base');
```

Comment:
We can specify where exactly we should find 'base' to speed up it
e.g. document.head.querySelector or document.body.querySelector

Reply:
1) `base` tag should be located in `head`, but this option can cover both `head` and `body`.
2) additionally it useless optimization, first of all, because `base` tag should be in `head`, which is the first tag in document.

Reply:
As I understood correctly, it should be always in the head
So I don't see any point to search it in both sources

Reply:
As I understood correctly, it should be always in the head
-- no, it can be in body too.